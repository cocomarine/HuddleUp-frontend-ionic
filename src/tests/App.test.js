import React from "react";
import { BrowserRouter } from "react-router-dom";
import { render, screen } from "@testing-library/react";
import { AuthContextProvider } from "../contexts/AuthContext";
import App from "../components/App";

describe("App", () => {
  xit("renders App correctly", () => {
    render(
      <BrowserRouter>
        <AuthContextProvider>
          <App />
        </AuthContextProvider>
      </BrowserRouter>
    );
    const app = screen.getByText(/huddleUp/i);

    expect(app).toBeInTheDocument();
  });
});
