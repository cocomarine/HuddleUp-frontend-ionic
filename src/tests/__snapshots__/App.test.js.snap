// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="homePg"
    >
      <div
        class="navbar"
      >
        <div
          class="leftSide"
        >
          <img
            alt="logo"
            class="l"
            src="huddleup_logo.jpeg"
            style="width: 100px; display: flex;"
          />
        </div>
        <div
          class="rightSide"
        >
          <div
            class="links"
            id=""
          >
            <a
              href="/"
            >
              Home
            </a>
            <a
              href="/login"
            >
              Login
            </a>
            <a
              href="/signup"
            >
              Sign up
            </a>
          </div>
        </div>
        <div
          class="icons"
        >
          <div
            class="closingIcon"
          />
          <div
            class="navIcon"
          >
            <svg
              color="rgb(246, 73, 73)"
              fill="currentColor"
              height="2.5rem"
              stroke="currentColor"
              stroke-width="0"
              style="color: rgb(246, 73, 73);"
              viewBox="0 0 24 24"
              width="2.5rem"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />
              <path
                d="M3 15h18v-2H3v2zm0 4h18v-2H3v2zm0-8h18V9H3v2zm0-6v2h18V5H3z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        class="homeBody"
      >
        <h1
          class="home-title"
        >
          <font
            color="#ED2643"
          >
            h
          </font>
          uddle
          <font
            color="#2284B5"
          >
            U
          </font>
          p
        </h1>
        <div
          class="homeText"
        >
          <h3
            style="font-weight: 200;"
          >
            Join us and plan your events and activities!
          </h3>
        </div>
        <div
          class="btns logged-out"
        >
          <button
            class="loginBtn"
          >
            Login
          </button>
          <button
            class="signUpBtn"
          >
            Sign up
          </button>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
